### ISSUE TITLE
partial-no-import fails for Google Font URL format

### ISSUE THREAD

**Issue by thegranddesign** [stylelint-scss/stylelint-scss#274]
Having:

```scss
@import 'https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700';
```

in a partial makes `partial-no-import` thrown an error when it should not.

**Comment by kristerkari** [comment]
Thanks! I'll have a look.

**Comment by kristerkari** [comment]
Sorry that I did not have time to look at this before now. @thegranddesign there is now a pull request with a fix for the issue #283. I'm going to release a new version with the fix soon.

**Comment by thegranddesign** [comment]
You’re awesome! Thank you!
